source/Base.h

```diff
@@ -124,6 +124,9 @@ template<typename... Ts>
 
 #define NOTSA_FORCEINLINE __forceinline
 
+//! An assert-like function that always executes the expression (Unlikes assert where it's lost in NDEBUG mode)
+#define NOTSA_AASSERT(_expr) do { bool res = !!(_expr); assert(res); } while(0);
```

Comment:
```suggestion
//! An assert function that works in the release mode as well.
#define NOTSA_AASSERT(_expr) do { bool res = !!(_expr); assert(res); } while(0);
```

i'd prefer naming this `CHECK` or sth, what do u think?

Reply:
Sounds too much like `VERIFY`, no?

Reply:
well actually removing this and making `VERIFY` assert in release makes more sense, cuz we check important stuff with it (e.g. win32 init stuff)

Reply:
actually shit the name is incorrect.
it doesn't `assert` in release mode, that's not the point.
it just executes whatever is inside the macro (unlike `assert` that is `#define assert(...) (void)(0)` in release mode)